Update vector_similarity_search.md (#25)
RedisInsight/Tutorials: src/redis_stack/vector_similarity_search.md

Redis Stack supports Vector Similarity Search - a new feature that is going to help to search unstructured data, like text and images. 

> Vector Similarity Search supports both hashes and JSON, but this tutorial describes examples for hashes only.

Reviewer: ```suggestion
Vector Similarity Search supports both hashes and JSON, but the examples in this tutorial only use hashes.
```
